content/en/tracing/setup_overview/setup/dotnet-core.md

```diff
@@ -211,97 +214,101 @@ export DD_DOTNET_TRACER_HOME=/opt/datadog
+</div>
 
-When using `systemctl` to run .NET applications as a service, you can also set environment variables to be loaded for all services ran via `systemctl`. To confirm these variables have been set, use [`systemctl show-environment`][3]. Before using this approach, see the note below about this instrumenting all .NET processes.
+When using `systemctl` to run .NET applications as a service, you can also set environment variables to be loaded for all services run by `systemctl`. 
 
-```bat
-systemctl set-environment CORECLR_ENABLE_PROFILING=1
-systemctl set-environment CORECLR_PROFILER={846F5F1C-F9AE-4B07-969E-05C26BC060D8}
-systemctl set-environment CORECLR_PROFILER_PATH=/opt/datadog/Datadog.Trace.ClrProfiler.Native.so
-systemctl set-environment DD_INTEGRATIONS=/opt/datadog/integrations.json
-systemctl set-environment DD_DOTNET_TRACER_HOME=/opt/datadog
-```
+1. Set the required environment variables by running [`systemctl set-environment`][2]:
+
+    ```bat
+    systemctl set-environment CORECLR_ENABLE_PROFILING=1
+    systemctl set-environment CORECLR_PROFILER={846F5F1C-F9AE-4B07-969E-05C26BC060D8}
+    systemctl set-environment CORECLR_PROFILER_PATH=/opt/datadog/Datadog.Trace.ClrProfiler.Native.so
+    systemctl set-environment DD_INTEGRATIONS=/opt/datadog/integrations.json
+    systemctl set-environment DD_DOTNET_TRACER_HOME=/opt/datadog
+    ```
+2. Verify that the environment variables were set by running `systemctl show-environment`.
 
-Once this is set, verify that the environment variables were set with `systemctl show-environment`.
+3. Restart the .NET service for the environment variables to take effect.
 
 [1]: /agent/guide/agent-configuration-files/#agent-main-configuration-file
-[2]: https://docs.docker.com/engine/reference/builder/#env
-[3]: https://www.freedesktop.org/software/systemd/man/systemd.exec.html#EnvironmentFile=
+[2]: https://www.freedesktop.org/software/systemd/man/systemctl.html#set-environment%20VARIABLE=VALUE%E2%80%A6
 {{% /tab %}}
 
 {{< /tabs >}}
 
-**Note:** The .NET runtime tries to load a profiler into _any_ .NET process that is started with these environment variables set. You should limit instrumentation only to the applications that need to be traced. **We do not recommend setting these environment variables globally as this causes _all_ .NET processes on the host to load the profiler.**
-
-### Configure the Datadog Agent for APM
-
-Install and configure the Datadog Agent to receive traces from your instrumented application. By default the Datadog Agent is enabled in your `datadog.yaml` file under `apm_enabled: true` and listens for trace traffic at `localhost:8126`. For containerized environments, follow the in-app [Quickstart instructions][3] to enable trace collection within the Datadog Agent.
```

Comment:
This links to https://docs.datadoghq.com/agent/basic_agent_usage/windows/, which is not "in-app Quickstart instructions." We either have the wrong link or the wrong description.